src/core/utils/strings.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
could we update resultfactory.js to use this truncate method as well?

Reply:
That all is going to get deleted in a future PR, so no need